Universal Tomb Launcher/Universal Tomb Launcher.csproj

```diff
@@ -8,9 +8,10 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace>UniversalTombLauncher</RootNamespace>
     <AssemblyName>launch</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>

```

Comment:
We cannot upgrade the version to 4.8 as this launcher is intended to work on systems as far back as Windows XP and on Windows 7 without having to manually install .NET 4, as it comes preinstalled with the system.